src/saturn_engine/worker/executors/queue.py

```diff
@@ -49,16 +48,16 @@ def start(self) -> None:
     async def run_queue(self) -> None:
         while self.is_running:
             processable = await self.queue.get()
-            processable.context.callback(self.queue.task_done)
+            processable._context.callback(self.queue.task_done)
```

Comment:
Why did you add the `_` at the beginning if you're using it outside of the class?

Reply:
I still consider it somewhat private, it just happens that the queue executor is closely coupled. I guess my thinking is that since `ExecutableMessage` is now pass to hooks, it's available to third-party code and I want to make it clear what is public API for these third-parties.

Reply:
Makes sense!